platforms/chibios/converters/promicro_to_rp2040_ce/_pin_defs.h

```diff
@@ -32,5 +32,5 @@
 #define B6 21U
 
 // LEDs
-#define D5 12U
-#define B0 13U
+#define D5 17U // user LED
+#define B0 25U // level shifted pin / RGB
```

Comment:
These seem helios specific, do they actually need to change?
```suggestion
#define D5 12U
#define B0 13U
```

Reply:
17 is defined as user led and 25 as RGB as per standard

Reply:
"standard" except half the existing boards do not have an LED on 17 and most are also missing RGB? Not much of a standard.

I have my reservations on mapping B0 to something tied to RGB, over the existing situation. The main aim is that legacy AVR code compiles, be it that it might not do anything. The proposal does still cover that in some aspects. However a board using RX/TX LEDs as indicator lights _might_ now cause the LED to show garbage.

Reply:
I understand your problem with this. So you would rather just expose them on the bottom row?